tests/test_ingest_pipeline.py

```diff
@@ -115,33 +117,31 @@ def test_ingest_cmd_missing_required_args():
 
 
 @pytest.mark.parametrize(
-    "cli_cmd, arg_list",
+    "cli_cmd, arg_list, expected_code",
     [
-        (cli.test, [SIMPLE_STUDY_CONFIG, "--log_level", "debug"]),
+        (cli.test, [SIMPLE_STUDY_CONFIG, "--no_validate"], 0),
+        (cli.test, [SIMPLE_STUDY_CONFIG, "--validation_mode", "advanced"], 0),
+        (cli.test, [SIMPLE_STUDY_CONFIG, "--log_level", "debug"], 0),
         (
             cli.ingest,
             [SIMPLE_STUDY_CONFIG, "--dry_run", "--log_level", "debug"],
+            0,
         ),
     ],
 )
-def test_ingest_cmds(cli_cmd, arg_list):
+def test_ingest_cmds(cli_cmd, arg_list, expected_code):
```

Comment:
Did you plan to have one that doesn't exit 0?

Reply:
Oh I think I changed this to take in an `expected_code` arg because before I had the ingest pipeline exit with code 1 when validation fails. Now the ingest pipeline will only exit with code 1 if something goes wrong in execution, so some sort of exception. So yea I could remove it since all exit codes are 0 right now